tests/native/test_manticore.py

```diff
@@ -88,6 +88,59 @@ def tmp(state):
 
         assert tmp in self.m._after_hooks[entry]
 
+    def test_add_sys_hook(self):
+        def tmp(state):
+            assert state._platformn._syscall_abi.syscall_number() == index
+            self.m.kill()
+
+        name = "sys_brk"
+        index = 12
```

Comment:
Nested functions in Python confuse me sometimes. Even though `index = 12` appears after line 93, `index` at line 93 will be `12` when it's executed? Maybe we should move `index = 12` to before the nested function like you have the test case after this one.